WordPress/src/main/res/values/strings.xml

```diff
@@ -615,6 +616,7 @@
     <string name="site_settings_whitelist_none_summary" translatable="false">@string/none</string>
     <string name="site_settings_optimize_images_summary">Enable to resize and compress pictures</string>
     <string name="site_settings_optimize_video_summary">Enable to resize and compress videos</string>
+    <string name="site_settings_enable_gutenberg_summary">This is still an experimental version of Gutenberg 🙈s</string>
```

Comment:
Is the `s` at the end there on purpose or is it a typo perhaps? 🤔

Reply:
Removed typo.